.github/container/Dockerfile.mjx

```diff
@@ -14,6 +15,11 @@ FROM ${BASE_IMAGE} as mealkit
 ARG SRC_PATH_MUJOCO
 ARG SRC_PATH_MUJOCO_MPC
 ARG SRC_PATH_L2R
+ARG CC
+ARG CXX
+
+ENV CC=${CC}
```

Comment:
Instead of enforcing the compiler toolchain globally, can we set it only for the MPC installation step?